src/main/java/com/hurko/controller/CardController.java

```diff
@@ -0,0 +1,62 @@
+package com.hurko.controller;
+
+import com.hurko.model.dto.Card;
+import com.hurko.service.CardService;
+import com.hurko.transformer.CardTransformer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.security.Principal;
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/card")
+public class CardController {
+
+    @Autowired
+    private CardService cardService;
+
+    @Autowired
+    private CardTransformer cardTransformer;
+
+    @PostMapping(path = "/put")
+    public void putMoney(@RequestParam int money, @RequestParam String number) {
```

Comment:
```suggestion
    public void putMoney(@RequestParam int money, @RequestParam String account) {
```
Maybe account is better variable name?